samples/server/petstore/jaxrs/jersey2/src/gen/java/io/swagger/api/FakeApi.java

```diff
@@ -120,13 +120,13 @@ public Response testClientModel(@ApiParam(value = "client model" ,required=true)
     
     @Consumes({ "application/xml; charset=utf-8", "application/json; charset=utf-8" })
     @Produces({ "application/xml; charset=utf-8", "application/json; charset=utf-8" })
-    @io.swagger.annotations.ApiOperation(value = "Fake endpoint for testing various parameters 假端點 偽のエンドポイント 가짜 엔드 포인트 ", notes = "Fake endpoint for testing various parameters 假端點 偽のエンドポイント 가짜 엔드 포인트 ", response = void.class, authorizations = {
+    @io.swagger.annotations.ApiOperation(value = "Fake endpoint for testing various parameters 假端點 偽のエンドポイント 가짜 엔드 포인트 ", notes = "Fake endpoint for testing various parameters 假端點 偽のエンドポイント 가짜 엔드 포인트 ", response = Void.class, authorizations = {
         @io.swagger.annotations.Authorization(value = "http_basic_test")
     }, tags={ "fake", })
     @io.swagger.annotations.ApiResponses(value = { 
-        @io.swagger.annotations.ApiResponse(code = 400, message = "Invalid username supplied", response = void.class),
+        @io.swagger.annotations.ApiResponse(code = 400, message = "Invalid username supplied", response = Void.class),
```

Comment:
Could you revert the formatting changes please.

Reply:
This was done on purpose as "void.class" is not a class in Java, so I updated it to "Void.class."

Reply:
@amcginn I think that might introduce a minor issue:
```
--- a/samples/server/petstore/jaxrs-cxf/src/gen/java/io/swagger/api/PetApi.java
+++ b/samples/server/petstore/jaxrs-cxf/src/gen/java/io/swagger/api/PetApi.java
@@ -32,7 +32,7 @@ public interface PetApi  {
     @ApiOperation(value = "Add a new pet to the store", tags={ "pet",  })
     @ApiResponses(value = { 
         @ApiResponse(code = 405, message = "Invalid input") })
-    public void addPet(@Valid Pet body);
+    public Void addPet(@Valid Pet body);
```
`void` becomes `Void` in the method signature and the code won't compile.

Reply:
I ran the jaxrs-cxf-petstore-server.sh while on master and this happened as well.